docs/src/js/ds6/main.js

```diff
@@ -1,9 +1,3 @@
-var FloatingLabel = require('makeup-floating-label');
-
-querySelectorAllToArray('.floating-label').forEach(function(el, i) {
-    var floatingLabel = new FloatingLabel(el);
-});
```

Comment:
Shouldn't this be here for the labels to work in the DS6 docs?

Reply:
I had moved this to the base/main.js since both docs now use it, and it is required in both dsx.browser.json files. Tested it locally as working in both DS4 and DS6 docs.

Reply:
Ah! I see now. Thanks.